### ISSUE TITLE
Background checks

### ISSUE THREAD

**Issue by benjaminwood** [OfficeMomsandDads/scheduler#17]
As a volunteer I want to complete my background check online so that I can save time/paperwork.

https://fortress.wa.gov/dshs/bcs/

11/13/18 UPDATE: I am told this link now works for background checks!

**Comment by benjaminwood** [comment]
@SarahOMD I see it says in the instructions:

> Have an opportunity to quickly email your name and confirmation number to the person or entity requesting the background check.

Are you familiar with this process? How do you envision the app's involvement in this process?

I think this will still be a post MVP feature, but I wanted to open this up for discussion so we can be ready when the time comes.

**Comment by SarahOMD** [comment]
I believe this is more for foster parent applicants or child care workers.  They are responsible to get the results of their background checks to their licensor/social worker.  In our case, it would be nice to have the results go directly to the Office Liaison and cut out the middle man (the applicant), but Im not sure that's an option if an applicant chooses this online version.  We could give them both options at the orientations I guess.

**Comment by benjaminwood** [comment]
Yeah, it'd be ideal if they offered a way for you to give people a specialized link to the form such that the system is aware of who is requesting and automatically sends them the results when complete. Might be worth asking 'them' about. Whoever they are :-)

**Comment by benjaminwood** [comment]
Determined no longer a priority.
